codes.md

```diff
@@ -70,3 +70,46 @@ Codes
+
+- [Session 006](https://mfr.osf.io/render?url=https://osf.io/pkqsm/?direct%26mode=render%26action=download%26mode=render) Envisioning one folder at time, what is inside a folder, doesn't explicitly mention "file system" until asked. "I only really consider one space at a time;" most of the structure is described from what they see from a location within the abstraction, rather than a bird's-eye view
+
+## Instances where participants drew something more like a node-link diagram, even though their description was more set-like
+
+- [Session 006](https://mfr.osf.io/render?url=https://osf.io/pkqsm/?direct%26mode=render%26action=download%26mode=render)
+- possibly [Session 010](https://mfr.osf.io/render?url=https://osf.io/kedv9/?direct%26mode=render%26action=download%26mode=render)
+
+## Instances where participants drew something more like an Euler diagram, even though their description was more tree-like
+
+- [Session 009](https://mfr.osf.io/render?url=https://osf.io/6n7pa/?direct%26mode=render%26action=download%26mode=render)
+
+## Instances where participants drew a table, even though their description was very non-tabular
+
+- [Session 014](https://mfr.osf.io/render?url=https://osf.io/dm3ak/?direct%26mode=render%26action=download%26mode=render)
+  - Surprised by their answer about whether their mental model changed (answer: kind of yes and no); feels like it contradicts how VIS people (or at least Alex) would interpret the way they first drew a "mind map," and then a table
+
+## Instances where participants primarily relied on text and labels, before, or in place of, visual encodings
+
+- [Session 010](https://mfr.osf.io/render?url=https://osf.io/kedv9/?direct%26mode=render%26action=download%26mode=render)
+- [Session 011](https://mfr.osf.io/render?url=https://osf.io/kaemj/?direct%26mode=render%26action=download%26mode=render)
+
+## Instances where participants relied on concrete pictures or glyphs of real-world objects instead of more abstract visual representations
+
+- [Session 007](https://mfr.osf.io/render?url=https://osf.io/hsy7q/?direct%26mode=render%26action=download%26mode=render)
+- [Session 008](https://mfr.osf.io/render?url=https://osf.io/47sng/?direct%26mode=render%26action=download%26mode=render)
+- [Session 006](https://mfr.osf.io/render?url=https://osf.io/pkqsm/?direct%26mode=render%26action=download%26mode=render)
+  - maybe questionable, depending on how we interpret their use of "manila folders"
+- [Session 013](https://mfr.osf.io/render?url=https://osf.io/dgzrs/?direct%26mode=render%26action=download%26mode=render)
+- [Session 015](https://mfr.osf.io/render?url=https://osf.io/y487f/?direct%26mode=render%26action=download%26mode=render)
+
+## Instances where participants used aggregative sketch marks to communicate the abstraction without having to draw the data in detail
+
```

Comment:
Also [Session 008](https://osf.io/wb4r5/) the participant deliberately decides to elide detail (instead of redrawing the power plant and cooling towers): "So D -- not gonna draw this over and over again."

Reply:
Oh, interesting that they didn't draw any aggregative marks at all; just extended the existing lines through where they imagined little pictures to be